i18n/en.json

```diff
@@ -15,7 +15,7 @@
 	"createwiki-error-notalnum": "The database name may not contain non-alphanumeric characters.",
 	"createwiki-error-notlowercase": "The database name may not contain uppercase letters.",
 	"createwiki-error-notsuffixed": "The database name doesn't end in a valid suffix (e.g. 'wiki').",
-	"createwiki-error-usernotcreated": "The requester user was not created on the wiki.",
+	"createwiki-error-usernotcreated": "The user that requested the wiki, was not created on the wiki.",
```

Comment:
There should probs be a space between to and the

Reply:
Huh?